README.md

```diff
@@ -0,0 +1,60 @@
+# Mountsion CBS Secondary Website
+
+![alt text](https://github.com/MountSionCBSSecondary/SchoolWebsite/tree/master/assets/images/Mount Sion Crest.jpg)
```

Comment:
This link is broke, if you rename the image to just `crest.jpg` then you can replace the line to `![school logo](assets/images/crest.jpg)`
Basically, you linked the full link which won't work you need to link the folder directory path from `README.md` to `crest.jpg`